Artesian/Artesian.SDK/Factory/ActualTimeSerie.cs

```diff
@@ -140,5 +141,36 @@ public async Task Save(Instant downloadedAt, bool deferCommandExecution = false,
                 await _marketDataService.UpsertCurveDataAsync(data, ctk);
             }
         }
+
+        /// <summary>
+        /// MarketData Delete
+        /// </summary>
+        /// <remarks>
+        /// Delete the Data of the current MarketData
+        /// </remarks>
+        /// <param name="rangeToDelete">Range to be deleted</param>
+        /// <param name="version">Version of the Time Serie</param>
+        /// <param name="product">Product of the MarketAssessment, BidAsk or Auction Time Serie</param>
+        /// <param name="deferCommandExecution">DeferCommandExecution</param>
+        /// <param name="deferDataGeneration">DeferDataGeneration</param>
+        /// <param name="keepNulls"></param>
+        /// <param name="ctk">The Cancellation Token</param> 
+        /// <returns></returns>
+        public async Task Delete(LocalDateTimeRange rangeToDelete, LocalDateTime? version = null, List<string> product = null, bool deferCommandExecution = false, bool deferDataGeneration = true, bool keepNulls = false, CancellationToken ctk = default)
```

Comment:
all the Query layer use 2 params for Ranges instead of the custom Ark.Tools range class to reduce dependencies.

Please keep the sdk consistent to user start/end pairs instead of Range.